openapi.yaml

```diff
@@ -4941,13 +4941,25 @@ paths:
     x-linode-cli-command: linodes
     post:
       x-linode-grant: read_write
-      summary: Initiate Pending Migration
+      summary: Initiate Pending Host Migration/DC Migration
       description: >
-        In some circumstances, a Linode may have pending migrations scheduled that
-        you can initiate when convenient.  In these cases, a Notification
-        will be returned from [GET /account/notifications](#getNotifications).
-        This endpoint initiates the scheduled migration, which will shut the Linode
-        down, migrate it, and then bring it back to its original state.
+        Initiate a pending host migration that has been scheduled by Linode or
+        initiate a cross data center (DC) migration.  A list of pending migrations,
+        if any, can be accessed from [GET /account/notifications](/api/v4/account-notifications).
+        When the migration begins, your Linode will be shutdown if not already off.
+        If the migration initiated the shutdown, it will reboot the Linode when completed.
+
+
+        To initiate a cross DC migration, you must pass a `region` parameter to the
+        request body specifying the target data center region.
+        You can view a list of all available regions and their feature capabilities
+        from [GET /regions](/api/v4/regions). If your Linode has a pending migration already queued,
+        you will not be able to initiate a DC migration until it has completed.
+
+
+        **Note:** Next Generation Network (NGN) data centers do not support IPv6 `/116` pools or IP Failover.
+        If you have these features enabled on your Linode and attempt to migrate to an NGN data center,
```

Comment:
@alsaihn can you confirm this? I thought we just wouldn't persist the IPv6 pool and IP failover, not that we would prevent the migration from happening. I did a quick check in the code and didn't see anything that would fail the migration.

Reply:
@yoshizzle I found that information in this related [KB post](https://kb.linode.com/pages/viewpage.action?pageId=42929211). I also checked w/ Sal, but more verification makes sense. Thanks!

Reply:
There's a ticket for that exact behavior in our backlog, so the comment as listed here is accurate for now. It will have to change later though.